server/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
```

Comment:
In most cases we do not want to remove starting year from copyright.
My understanding is that even if we copied then modified another file, the result is a modification, and would generally maintain the chain of copyright claims.
It _might_ be OK in this case, but it's probably safer to always keep the first year and only ever update the second (aside from brand new files, of course).

Reply:
If spotless is removing earlier years, then we need to fix the copyright check in spotless (which we do control, it's a block in a gradle file).

Reply:
✅ Done. Reverted.